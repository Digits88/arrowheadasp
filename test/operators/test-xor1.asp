Test logical XOR operator:
<hr>

True xor True =         <%= True xor True %><br>
True xor False =        <%= True xor False %><br>
IsNull(True xor Null) = <%= IsNull(True xor Null) %><br>
False xor True =        <%= False xor True %><br>
False xor False =       <%= False xor False %><br>
IsNull(False xor Null) =<%= IsNull(False xor Null) %><br>
IsNull(Null xor True) = <%= IsNull(Null xor True) %><br>
IsNull(Null xor False) =<%= IsNull(Null xor False) %><br>
IsNull(Null xor Null) = <%= IsNull(Null xor Null) %><br>
1 xor 1 =               <%= 1 xor 1 %><br>
&amp;HFF xor &amp;H10 = <%= &HFF xor &H10 %><br>
&amp;H33 xor &amp;H11 = <%= &H33 xor &H11 %><br>
IsNull(Null xor &amp;HFF) =     <%= IsNull(NULL xor &HFF) %><br>
IsNull(Null xor &amp;H01) =     <%= IsNull(NULL xor &H01) %><br>
IsNull(Null xor &amp;H00) =     <%= IsNull(NULL xor &H00) %><br>
&amp;11 xor TRUE = <%= &H11 xor TRUE %><br>
&amp;11 xor FALSE = <%= &H11 xor FALSE %><br>
TRUE xor &amp;11 = <%= TRUE xor &H11 %><br>
FALSE xor &amp;11 = <%= FALSE xor &H11 %><br>

