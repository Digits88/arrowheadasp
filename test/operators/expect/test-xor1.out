Test logical XOR operator:
<hr>

True xor True =         False<br>
True xor False =        True<br>
IsNull(True xor Null) = True<br>
False xor True =        True<br>
False xor False =       False<br>
IsNull(False xor Null) =True<br>
IsNull(Null xor True) = True<br>
IsNull(Null xor False) =True<br>
IsNull(Null xor Null) = True<br>
1 xor 1 =               0<br>
&amp;HFF xor &amp;H10 = 239<br>
&amp;H33 xor &amp;H11 = 34<br>
IsNull(Null xor &amp;HFF) =     True<br>
IsNull(Null xor &amp;H01) =     True<br>
IsNull(Null xor &amp;H00) =     True<br>
&amp;11 xor TRUE = -18<br>
&amp;11 xor FALSE = 17<br>
TRUE xor &amp;11 = -18<br>
FALSE xor &amp;11 = 17<br>

